roles/dhcp/defaults/main.yml

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
These example entries should go in the `test` directory as they aren't default values to be used in the role